<!doctype html>
<head>
</head>
</body>
<div id="readme" class="readme blob instapaper_body">
    <article class="markdown-body entry-content" itemprop="text"><p>For biology students, I think learning a <a href="https://en.wikipedia.org/wiki/Programming_language">programming language</a> is not a waste of time. However, there is many, many different languages [<a href="https://en.wikipedia.org/wiki/List_of_programming_languages">Wiki</a>] and you have to choose a language with a good balance between criteria like ease of learning, wealth of tutorials and examples, large user community,etc. Nowadays, IMHO, two programming languages are fulfilling these criteria:</p>

<ul>
<li>Python </li>
<li>JavaScript and all the web technologies (HTML5 and CSS3)</li>
</ul>

<blockquote>
<p><em>Note</em>*: If you need more sophisticated statistics functions, it is good to look at the "R" language. It is more complex but more powerful in this field.</p>
</blockquote>

<h2><a id="user-content-1--python-programming-language" class="anchor" href="#1--python-programming-language" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1- Python programming language</h2>

<h3><a id="user-content-1-1--the-basics-variables-conditionals-and-loops" class="anchor" href="#1-1--the-basics-variables-conditionals-and-loops" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-1- The Basics: Variables, conditionals, and loops</h3>

<h4><a id="user-content-1-1-1-variables" class="anchor" href="#1-1-1-variables" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-1-1 Variables</h4>

<p>Variables are <em>"boxes"</em> containing one value. This value may be a:</p>

<ul>
<li>Number (integer, floating-point numbers,etc.)</li>
<li>Boolean (<strong>True</strong> of <strong>False</strong>)</li>
<li>String (aka Text)</li>
<li>List</li>
<li>Dictionary</li>
<li>Object</li>
</ul>

<p>Variables names are case sensitive, cannot begin with a digit and you cannot use a reserved keyword used by Python (like <strong>for</strong>).</p>

<div class="highlight highlight-source-python"><pre>myVar <span class="pl-k">=</span> <span class="pl-c1">3</span>
myVar <span class="pl-k">=</span> myVar <span class="pl-k">+</span> <span class="pl-c1">1</span> <span class="pl-c"># ← 4</span>

<span class="pl-ii">0test</span> <span class="pl-k">=</span> <span class="pl-c1">5</span> <span class="pl-c"># ← ERROR</span>
</pre></div>

<h4><a id="user-content-1-1-2-conditionals" class="anchor" href="#1-1-2-conditionals" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-1-2 Conditionals</h4>

<div class="highlight highlight-source-python"><pre><span class="pl-k">if</span> condition :
  instruction_1
  instruction_2
  instruction_3
<span class="pl-k">elif</span> other_condition:
  instruction_a
  instruction_b
<span class="pl-k">else</span>:
  instruction_01
  instruction_02
  instruction_03
  instruction_04</pre></div>

<h4><a id="user-content-1-1-3-loops" class="anchor" href="#1-1-3-loops" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-1-3 Loops</h4>

<h5><a id="user-content-1-1-3-1-while-loop" class="anchor" href="#1-1-3-1-while-loop" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-1-3-1 <strong>While</strong> Loop</h5>

<div class="highlight highlight-source-python"><pre><span class="pl-k">while</span> exit_condition:
  instruction_1
  instruction_2
  instruction_3
</pre></div>

<p>For example, to display each character of the string 'python'</p>

<div class="highlight highlight-source-python"><pre>lang <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>python<span class="pl-pds">'</span></span>
i <span class="pl-k">=</span> <span class="pl-c1">0</span>
<span class="pl-k">while</span> i <span class="pl-k">&lt;</span> <span class="pl-c1">len</span>(lang):
  <span class="pl-c1">print</span>(lang[i])
  i <span class="pl-k">=</span> i <span class="pl-k">+</span> <span class="pl-c1">1</span></pre></div>

<p>And the result is...</p>

<div class="highlight highlight-source-python"><pre>p
y
t
h
o
n</pre></div>

<h5><a id="user-content-1-1-3-2-for-in-loop" class="anchor" href="#1-1-3-2-for-in-loop" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-1-3-2 <strong>for in</strong> Loop</h5>

<p>This loop requires a list and apply the block of instructions for each element with the following syntax:</p>

<div class="highlight highlight-source-python"><pre><span class="pl-k">for</span> obj <span class="pl-k">in</span> a_list:
  instruction_1
  instruction_2
  instruction_3
</pre></div>

<p>Here is a basic example...</p>

<div class="highlight highlight-source-python"><pre>lang <span class="pl-k">=</span> [<span class="pl-s"><span class="pl-pds">'</span>p<span class="pl-pds">'</span></span>,<span class="pl-s"><span class="pl-pds">'</span>y<span class="pl-pds">'</span></span>,<span class="pl-s"><span class="pl-pds">'</span>t<span class="pl-pds">'</span></span>,<span class="pl-s"><span class="pl-pds">'</span>h<span class="pl-pds">'</span></span>,<span class="pl-s"><span class="pl-pds">'</span>o<span class="pl-pds">'</span></span>,<span class="pl-s"><span class="pl-pds">'</span>n<span class="pl-pds">'</span></span>]

<span class="pl-k">for</span> char <span class="pl-k">in</span> lang:
  <span class="pl-c1">print</span>(char)</pre></div>

<p>This type of loop is very often used with the <em>range(...)</em> function. This very convenient function <em>range(star, end, step)</em> creates a list containing a series of numbers.</p>

<div class="highlight highlight-source-python"><pre>mySeries <span class="pl-k">=</span> <span class="pl-c1">range</span>(<span class="pl-c1">0</span>,<span class="pl-c1">10</span>,<span class="pl-c1">2</span>) <span class="pl-c">#  ← [0,2,4,6,8]</span>

<span class="pl-k">for</span> i <span class="pl-k">in</span> mySeries:
  <span class="pl-c1">print</span>(i)

movie <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>starwarsreturnofthejedi.<span class="pl-pds">'</span></span>
<span class="pl-k">for</span> i <span class="pl-k">in</span> <span class="pl-c1">range</span>(<span class="pl-c1">1</span>, <span class="pl-c1">10</span>):
  poweroftwo <span class="pl-k">=</span> i <span class="pl-k">**</span> <span class="pl-c1">2</span>
  <span class="pl-c1">print</span>(poweroftwo) <span class="pl-c"># 1, 4, 9, 16, 25, ..., 100</span>
</pre></div>

<h3><a id="user-content-1-2--advanced-functions" class="anchor" href="#1-2--advanced-functions" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-2- Advanced: Functions</h3>

<div class="highlight highlight-source-python"><pre><span class="pl-k">def</span> <span class="pl-en">function_name</span>(<span class="pl-smi">arg1</span>,<span class="pl-smi">arg2</span>,..., <span class="pl-smi">argn</span>):
  instruction_1
  instruction_2
  instruction_3
  <span class="pl-k">return</span> result</pre></div>

<p>For example, if you want to get codons of a nucleic sequence ...</p>

<div class="highlight highlight-source-python"><pre><span class="pl-k">def</span> <span class="pl-en">getCodons</span>(<span class="pl-smi">seq</span>):
  <span class="pl-s"><span class="pl-pds">"""</span> Extract codons from a nucleic sequence <span class="pl-pds">"""</span></span>
  codons_list <span class="pl-k">=</span> []
  <span class="pl-k">for</span> i <span class="pl-k">in</span> <span class="pl-c1">range</span>(<span class="pl-c1">0</span>,<span class="pl-c1">len</span>(seq),<span class="pl-c1">3</span>):
    codon <span class="pl-k">=</span> seq[i:i<span class="pl-k">+</span><span class="pl-c1">3</span>]
    codons_list.append(codon)
  <span class="pl-k">return</span> codons_list

<span class="pl-c"># Test</span>
mySeq<span class="pl-k">=</span><span class="pl-s"><span class="pl-pds">'</span>actgctgtcgaaccg<span class="pl-pds">'</span></span>
myCodons <span class="pl-k">=</span> getCodons(mySeq) <span class="pl-c"># ← ['act', 'gct', 'gtc', 'gaa', 'ccg']</span></pre></div>

<h3><a id="user-content-1-2--learning-python-with-web-based-tool" class="anchor" href="#1-2--learning-python-with-web-based-tool" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-2- Learning Python with web-based tool</h3>

<p><a href="http://pythonfiddle.com/">Python Fiddle</a> is a good starting environment to write your first scripts.</p>

<h3><a id="user-content-1-3--learning-python-with-idle" class="anchor" href="#1-3--learning-python-with-idle" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>1-3- Learning Python with IDLE</h3>

<h2><a id="user-content-2--python-helper-code" class="anchor" href="#2--python-helper-code" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>2- Python helper Code</h2>

<p>Using extensively Python Fiddle, it is not possible AFAIK to import BioPython. Moreover, the latter is a little bit too complex for biology students with no programming skills.</p>

<p><strong>Seq</strong> class storing information (title and sequence data) from a FASTA sequence.</p>

<div class="highlight highlight-source-python"><pre><span class="pl-k">class</span> <span class="pl-en">Seq</span>:
    <span class="pl-k">def</span> <span class="pl-c1">__init__</span>(<span class="pl-smi"><span class="pl-smi">self</span></span>, <span class="pl-smi">seqdata</span>):
        _tmp <span class="pl-k">=</span> seqdata.split(<span class="pl-s"><span class="pl-pds">'</span><span class="pl-cce">\n</span><span class="pl-pds">'</span></span>)
        <span class="pl-v">self</span>.description <span class="pl-k">=</span> _tmp[<span class="pl-c1">0</span>][<span class="pl-c1">1</span>:] <span class="pl-k">if</span> _tmp[<span class="pl-c1">0</span>][<span class="pl-c1">0</span>] <span class="pl-k">==</span> <span class="pl-s"><span class="pl-pds">'</span>&gt;<span class="pl-pds">'</span></span> <span class="pl-k">else</span> _tmp[<span class="pl-c1">0</span>]
        <span class="pl-v">self</span>.data <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span><span class="pl-pds">'</span></span>.join(_tmp[<span class="pl-c1">1</span>:]).strip()

        <span class="pl-c"># Read title</span>
        <span class="pl-v">self</span>.author1 <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>
        <span class="pl-v">self</span>.author2 <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>
        <span class="pl-v">self</span>.copy    <span class="pl-k">=</span> <span class="pl-c1">0</span>
        <span class="pl-v">self</span>.db      <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>
        <span class="pl-v">self</span>.id      <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>
        <span class="pl-v">self</span>.db2     <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>
        <span class="pl-v">self</span>.acc     <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>
        <span class="pl-v">self</span>.title   <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>None<span class="pl-pds">'</span></span>

        <span class="pl-c"># Try to read information within the description</span>
        sep <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">'</span>|<span class="pl-pds">'</span></span>
        _tmp <span class="pl-k">=</span> <span class="pl-v">self</span>.description.split(sep)
        <span class="pl-v">self</span>.db <span class="pl-k">=</span> _tmp[<span class="pl-c1">0</span>]
        <span class="pl-k">if</span> <span class="pl-v">self</span>.db <span class="pl-k">==</span> <span class="pl-s"><span class="pl-pds">'</span>xzy<span class="pl-pds">'</span></span>:
            <span class="pl-c">#CrazyBio header: xzy|first author|copynumber|second author</span>
            <span class="pl-v">self</span>.author1 <span class="pl-k">=</span> _tmp[<span class="pl-c1">1</span>]
            <span class="pl-v">self</span>.author2 <span class="pl-k">=</span> _tmp[<span class="pl-c1">3</span>]
            <span class="pl-v">self</span>.copy    <span class="pl-k">=</span> <span class="pl-c1">int</span>( _tmp[<span class="pl-c1">2</span>])
            <span class="pl-v">self</span>.length  <span class="pl-k">=</span> <span class="pl-c1">len</span>(<span class="pl-v">self</span>.data)
        <span class="pl-k">elif</span> <span class="pl-v">self</span>.db <span class="pl-k">==</span> <span class="pl-s"><span class="pl-pds">'</span>gi<span class="pl-pds">'</span></span>:
            <span class="pl-c"># gi|numéro gi|gb|numéro d'accession|locus</span>
            <span class="pl-v">self</span>.id   <span class="pl-k">=</span> _tmp[<span class="pl-c1">1</span>]
            <span class="pl-v">self</span>.db2  <span class="pl-k">=</span> _tmp[<span class="pl-c1">2</span>]
            <span class="pl-v">self</span>.acc  <span class="pl-k">=</span> _tmp[<span class="pl-c1">3</span>]
            <span class="pl-v">self</span>.title <span class="pl-k">=</span> _tmp[<span class="pl-c1">4</span>]
        <span class="pl-k">elif</span> <span class="pl-v">self</span>.db <span class="pl-k">==</span> <span class="pl-s"><span class="pl-pds">'</span>sp<span class="pl-pds">'</span></span>:
            <span class="pl-c"># sp|numéro d'accession|nom</span>
            <span class="pl-v">self</span>.db   <span class="pl-k">=</span> _tmp[<span class="pl-c1">0</span>]
            <span class="pl-v">self</span>.acc  <span class="pl-k">=</span> _tmp[<span class="pl-c1">1</span>]
            <span class="pl-v">self</span>.title <span class="pl-k">=</span> _tmp[<span class="pl-c1">2</span>]

    <span class="pl-k">def</span> <span class="pl-en">show</span>(<span class="pl-smi"><span class="pl-smi">self</span></span>):
        attrs <span class="pl-k">=</span> <span class="pl-c1">vars</span>(<span class="pl-v">self</span>)
        <span class="pl-k">return</span> <span class="pl-s"><span class="pl-pds">'</span>, <span class="pl-pds">'</span></span>.join(<span class="pl-s"><span class="pl-pds">"</span><span class="pl-c1">%s</span>: <span class="pl-c1">%s</span><span class="pl-pds">"</span></span> <span class="pl-k">%</span> item <span class="pl-k">for</span> item <span class="pl-k">in</span> attrs.items())

    <span class="pl-k">def</span> <span class="pl-en">fasta</span>(<span class="pl-smi"><span class="pl-smi">self</span></span>):
        <span class="pl-k">return</span> <span class="pl-s"><span class="pl-pds">'</span>&gt;<span class="pl-c1">{<span class="pl-c1">:s</span>}</span><span class="pl-cce">\n</span><span class="pl-c1">{<span class="pl-c1">:s</span>}</span><span class="pl-pds">'</span></span>.format(<span class="pl-v">self</span>.description,<span class="pl-v">self</span>.data)
</pre></div>

<p>Usage</p>

<div class="highlight highlight-source-python"><pre>fasta <span class="pl-k">=</span> <span class="pl-s"><span class="pl-pds">"""</span>&gt;sp|P68871|HBB_HUMAN Hemoglobin subunit beta OS=Homo sapiens GN=HBB PE=1 SV=2</span>
<span class="pl-s">MVHLTPEEKSAVTALWGKVNVDEVGGEALGRLLVVYPWTQRFFESFGDLSTPDAVMGNPK</span>
<span class="pl-s">VKAHGKKVLGAFSDGLAHLDNLKGTFATLSELHCDKLHVDPENFRLLGNVLVCVLAHHFG</span>
<span class="pl-s">KEFTPPVQAAYQKVVAGVANALAHKYH<span class="pl-pds">"""</span></span>

seq <span class="pl-k">=</span> Seq(fasta)
<span class="pl-c1">print</span> seq.acc <span class="pl-c"># &amp;larr; P68871</span></pre></div>

<h2><a id="user-content-3--list-of-emboss-tools" class="anchor" href="#3--list-of-emboss-tools" aria-hidden="true"><svg aria-hidden="true" class="octicon octicon-link" height="16" version="1.1" viewBox="0 0 16 16" width="16"><path d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>3- List of EMBOSS Tools</h2>

<p><a href="http://emboss.sourceforge.net/">EMBOSS</a> — European Molecular Biology Open Software Suite — is a European package containing various bioinformatics programs available as web-based or as local tools.
The list of all the tools is available <a href="http://emboss.sourceforge.net/apps/release/6.6/emboss/apps/">here</a> and grouped by categories <a href="http://emboss.sourceforge.net/apps/release/6.6/emboss/apps/groups.html">there</a> and I created my own version combining both <a href="http://crazybiocomputing.github.io/list_emboss.html">there</a> </p>
</article>
  </div>

</div>
</body>
</html>
