<!DOCTYPE html>
<html>
<head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<meta name="Distribution" content="Global" />
<meta name="Author" content="Jean-Christophe Taveau" />
<meta name="Robots" content="index,follow" />

<link rel="shortcut icon" href="/images/crazybio.ico" />

<title>CrazyBioComputing</title>
<link href='https://fonts.googleapis.com/css?family=Rock+Salt' rel='stylesheet' type='text/css'>
<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
<link href='crazybio.css' rel='stylesheet' type='text/css'>

<style>

.group_title {
  font-family: 'Arial';
  font-weight: bold;
  color: white;
  background-color: #000;
  margin:10px 0px 5px 0px;
  width: 800px;
}

.group {
  font-family: 'Arial';
  margin:2px 0px 20px 0px;
  display:table;
}

.row {
    list-style:none;
    display:table-row-group;
    margin-left: 20px;
}

.row li {
 display:table-cell;
 padding-right: 10px;
}


.row li:first-child {
  font-weight: bold;
}

.application {
   padding: 0px 40px 0px 5px;
   font-weight: bold;
}

.description {
   padding: 0px 40px 0px 5px;
   float:right;
}



</style>

</head>


<body>
 <!-- HEADER -->
<div id="header_wrap" class="outer">
<header class="inner">
<h1 id="project_title">Crazybiocomputing.GitHub.io</h1>
<h2 id="project_tagline">Tools about Bioinformatics</h2>
</header>
</div>
<!-- MAIN CONTENT -->
<div id="main_content_wrap" class="outer">
<section id="main_content" class="inner">
<h3>
<a id="welcome-to-github-pages" class="anchor" href="#welcome-to-github-pages" aria-hidden="true"><span class="octicon octicon-link"></span></a>Welcome to GitHub Pages.</h3>

<h1 style="font-family: 'Rock Salt', cursive;"> CrazyBioComputing- BioInformatics</h1>

<div id="nav">
<ul>
<li style="font-family: 'Rock Salt', cursive;"><a href="index.html">Home</a></li>
<li style="font-family: 'Rock Salt', cursive;">Bioinformatics</li>
<li style="font-family: 'Rock Salt', cursive;">GIS</a></li>
<li style="font-family: 'Rock Salt', cursive;">Image Processing</a></li>
<li style="font-family: 'Rock Salt', cursive;"><a href="http://crazybiocomputing.blogspot.com/">Blog</a></li>
<li style="font-family: 'Rock Salt', cursive;">Quizz</a></li>
<li style="font-family: 'Rock Salt', cursive;">Tools</a></li>
</ul>
</div>


<div id="page">
  <div id="left">
    <ul>
      <li>Mini-Games</li>
      <li>Resources</li>
      <li>Tools</li>
    </ul>
  </div>

  <div id="content">
<h2>List of EMBOSS applications</h2>

<p>By Jean-Christophe Taveau.</p>
<p>A small utility page to easily search an application in the package. This page was calculated from release 6.6.</p>
 
<h3> 1- Applications of release 6.6</h3>
<p>This list was created from the following genuine links...<br>
</p>
 

 
<ul>
<li>Applications sort by groups <a href="http://emboss.sourceforge.net/apps/release/6.6/emboss/apps/groups.html">[L i n k]</a>
</li>
<li>Applications sort alphabetically <a href="http://emboss.sourceforge.net/apps/release/6.6/emboss/apps/index.html">[L i n k]</a>
</li></ul>
 


  
<h3> 2- List of applications with groups</h3>
 
<ul>
<li class="group_title">Acd &#8212; Acd file utilities</li>
<ul class="group">
<ul class="row"><li>acdc</li><li>Test an application ACD file</li></ul>
<ul class="row"><li>acdpretty</li><li>Correctly reformat an application ACD file</li></ul>
<ul class="row"><li>acdtable</li><li>Generate an HTML table of parameters from an application ACD file</li></ul>
<ul class="row"><li>acdtrace</li><li>Trace processing of an application ACD file (for testing)</li></ul>
<ul class="row"><li>acdvalid</li><li>Validate an application ACD file</li></ul>
</ul>
<li class="group_title">Alignment &#8212; Sequence comparison and alignment</li>
<ul class="group">
<ul class="row"><li>extractalign</li><li>Extract regions from a sequence alignment </li></ul>
</ul>
<li class="group_title">Alignment consensus &#8212; Merging sequences to make a consensus</li>
<ul class="group">
<ul class="row"><li>cons</li><li>Create a consensus sequence from a multiple alignment</li></ul>
<ul class="row"><li>consambig</li><li>Create an ambiguous consensus sequence from a multiple alignment</li></ul>
<ul class="row"><li>megamerger</li><li>Merge two large overlapping DNA sequences</li></ul>
<ul class="row"><li>merger</li><li>Merge two overlapping sequences </li></ul>
</ul>
<li class="group_title">Alignment differences &#8212; Finding differences between sequences</li>
<ul class="group">
<ul class="row"><li>diffseq</li><li>Compare and report features of two similar sequences </li></ul>
</ul>
<li class="group_title">Alignment dot plots &#8212; Dot plot sequence comparisons</li>
<ul class="group">
<ul class="row"><li>dotmatcher</li><li>Draw a threshold dotplot of two sequences</li></ul>
<ul class="row"><li>dotpath</li><li>Draw a non-overlapping wordmatch dotplot of two sequences</li></ul>
<ul class="row"><li>dottup</li><li>Display a wordmatch dotplot of two sequences</li></ul>
<ul class="row"><li>polydot</li><li>Draw dotplots for all-against-all comparison of a sequence set </li></ul>
</ul>
<li class="group_title">Alignment global &#8212; Global sequence alignment</li>
<ul class="group">
<ul class="row"><li>est2genome</li><li>Align EST sequences to genomic DNA sequence</li></ul>
<ul class="row"><li>needle</li><li>Needleman-Wunsch global alignment of two sequences</li></ul>
<ul class="row"><li>needleall</li><li>Many-to-many pairwise alignments of two sequence sets</li></ul>
<ul class="row"><li>stretcher</li><li>Needleman-Wunsch rapid global alignment of two sequences</li></ul>
<ul class="row"><li>esim4</li><li>Align an mRNA to a genomic DNA sequence </li></ul>
</ul>
<li class="group_title">Alignment local &#8212; Local sequence alignment</li>
<ul class="group">
<ul class="row"><li>matcher</li><li>Waterman-Eggert local alignment of two sequences</li></ul>
<ul class="row"><li>seqmatchall</li><li>All-against-all word comparison of a sequence set</li></ul>
<ul class="row"><li>supermatcher</li><li>Calculate approximate local pair-wise alignments of larger sequences</li></ul>
<ul class="row"><li>water</li><li>Smith-Waterman local alignment of sequences</li></ul>
<ul class="row"><li>wordfinder</li><li>Match large sequences against one or more other sequences</li></ul>
<ul class="row"><li>wordmatch</li><li>Find regions of identity (exact matches) of two sequences</li></ul>
<ul class="row"><li>domainalign</li><li>Generate alignments (DAF file) for nodes in a DCF file</li></ul>
<ul class="row"><li>domainrep</li><li>Reorder DCF file to identify representative structures</li></ul>
<ul class="row"><li>seqalign</li><li>Extend alignments (DAF file) with sequences (DHF file)</li></ul>
<ul class="row"><li>oalistat</li><li>Statistics for multiple alignment files </li></ul>
</ul>
<li class="group_title">Alignment multiple &#8212;Multiple sequence alignment</li>
<ul class="group">
<ul class="row"><li>edialign</li><li>Local multiple alignment of sequences</li></ul>
<ul class="row"><li>emma</li><li>Multiple sequence alignment (ClustalW wrapper)</li></ul>
<ul class="row"><li>infoalign</li><li>Display basic information about a multiple sequence alignment</li></ul>
<ul class="row"><li>plotcon</li><li>Plot conservation of a sequence alignment</li></ul>
<ul class="row"><li>prettyplot</li><li>Draw a sequence alignment with pretty formatting</li></ul>
<ul class="row"><li>showalign</li><li>Display a multiple sequence alignment in pretty format</li></ul>
<ul class="row"><li>tranalign</li><li>Generate an alignment of nucleic coding regions from aligned proteins</li></ul>
<ul class="row"><li>eomega</li><li>Multiple sequence alignment (ClustalO wrapper)</li></ul>
<ul class="row"><li>eomegapp</li><li>Profile with profile (ClustalO wrapper)</li></ul>
<ul class="row"><li>eomegash</li><li>Sequence with HMM (ClustalO wrapper)</li></ul>
<ul class="row"><li>eomegasp</li><li>Sequence with profile (ClustalO wrapper)</li></ul>
<ul class="row"><li>mse</li><li>Multiple sequence editor </li></ul>
</ul>
<li class="group_title">Assembly fragment assembly &#8212; DNA sequence assembly</li>
<ul class="group">
<ul class="row"><li>emira</li><li>MIRA fragment assembly program</li></ul>
<ul class="row"><li>emiraest</li><li>MIRAest fragment assembly program </li></ul>
</ul>
<li class="group_title">Data resources &#8212; Data resources</li>
<ul class="group">
<ul class="row"><li>cachedas</li><li>Generate server cache file for DAS servers or for the DAS registry</li></ul>
<ul class="row"><li>cachedbfetch</li><li>Generate server cache file for Dbfetch/WSDbfetch data sources</li></ul>
<ul class="row"><li>cacheebeyesearch</li><li>Generate server cache file for EB-eye search domains</li></ul>
<ul class="row"><li>cacheensembl</li><li>Generate server cache file for an Ensembl server</li></ul>
<ul class="row"><li>dbtell</li><li>Display information about a public database</li></ul>
<ul class="row"><li>servertell</li><li>Display information about a public server</li></ul>
<ul class="row"><li>showdb</li><li>Display information on configured databases</li></ul>
<ul class="row"><li>showserver</li><li>Display information on configured servers </li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Data retrieval</li>
<ul class="group">
<ul class="row"><li>codcopy</li><li>Copy and reformat a codon usage table</li></ul>
<ul class="row"><li>seqret</li><li>Read and write (return) sequences </li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Chemistry data retrieval</li>
<ul class="group">
<ul class="row"><li>infobase</li><li>Return information on a given nucleotide base</li></ul>
<ul class="row"><li>inforesidue</li><li>Return information on a given amino acid residue </li></ul>
</ul>
<li class="group_title">Data retrieval &#8212;  Sequence feature data retrieval</li>
<ul class="group">
<ul class="row"><li>featcopy</li><li>Read and write a feature table</li></ul>
<ul class="row"><li>featmerge</li><li>Merge two overlapping feature tables </li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Ontology data retrieval</li>
<ul class="group">
<ul class="row"><li>edamdef</li><li>Find EDAM ontology terms by definition</li></ul>
<ul class="row"><li>edamhasinput</li><li>Find EDAM ontology terms by has_input relation</li></ul>
<ul class="row"><li>edamhasoutput</li><li>Find EDAM ontology terms by has_output relation</li></ul>
<ul class="row"><li>edamisformat</li><li>Find EDAM ontology terms by is_format_of relation</li></ul>
<ul class="row"><li>edamisid</li><li>Find EDAM ontology terms by is_identifier_of relation</li></ul>
<ul class="row"><li>edamname</li><li>Find EDAM ontology terms by name</li></ul>
<ul class="row"><li>godef</li><li>Find GO ontology terms by definition</li></ul>
<ul class="row"><li>goname</li><li>Find GO ontology terms by name</li></ul>
<ul class="row"><li>ontoget</li><li>Get ontology term(s)</li></ul>
<ul class="row"><li>ontogetcommon</li><li>Get common ancestor for terms</li></ul>
<ul class="row"><li>ontogetdown</li><li>Get ontology term(s) by parent id</li></ul>
<ul class="row"><li>ontogetobsolete</li><li>Get ontology ontology terms</li></ul>
<ul class="row"><li>ontogetroot</li><li>Get ontology root terms by child identifier</li></ul>
<ul class="row"><li>ontogetsibs</li><li>Get ontology term(s) by id with common parent</li></ul>
<ul class="row"><li>ontogetup</li><li>Get ontology term(s) by id of child</li></ul>
<ul class="row"><li>ontoisobsolete</li><li>Report whether an ontology term id is obsolete</li></ul>
<ul class="row"><li>ontotext</li><li>Get ontology term(s) original full text</li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Resource data retrieval</li>
<ul class="group">
<ul class="row"><li>drfinddata</li><li>Find public databases by data type</li></ul>
<ul class="row"><li>drfindformat</li><li>Find public databases by format</li></ul>
<ul class="row"><li>drfindid</li><li>Find public databases by identifier</li></ul>
<ul class="row"><li>drfindresource</li><li>Find public databases by resource</li></ul>
<ul class="row"><li>drget</li><li>Get data resource entries</li></ul>
<ul class="row"><li>drtext</li><li>Get data resource entries complete text</li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Sequence data retrieval</li>
<ul class="group">
<ul class="row"><li>abiview</li><li>Display the trace in an ABI sequencer file</li></ul>
<ul class="row"><li>coderet</li><li>Extract CDS, mRNA and translations from feature tables</li></ul>
<ul class="row"><li>entret</li><li>Retrieve sequence entries from flatfile databases and files</li></ul>
<ul class="row"><li>extractalign</li><li>Extract regions from a sequence alignment</li></ul>
<ul class="row"><li>infoalign</li><li>Display basic information about a multiple sequence alignment</li></ul>
<ul class="row"><li>infoseq</li><li>Display basic information about sequences</li></ul>
<ul class="row"><li>refseqget</li><li>Get reference sequence</li></ul>
<ul class="row"><li>seqxref</li><li>Retrieve all database cross-references for a sequence entry</li></ul>
<ul class="row"><li>seqxrefget</li><li>Retrieve all cross-referenced data for a sequence entry</li></ul>
<ul class="row"><li>showalign</li><li>Display a multiple sequence alignment in pretty format</li></ul>
<ul class="row"><li>variationget</li><li>Get sequence variations</li></ul>
<ul class="row"><li>whichdb</li><li>Search all sequence databases for an entry and retrieve it</li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Sequence assembly data retrieval</li>
<ul class="group">
<ul class="row"><li>assemblyget</li><li>Get assembly of sequence reads</li></ul>
<ul class="row"><li>infoassembly</li><li>Display information about assemblies</li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Text data retrieval</li>
<ul class="group">
<ul class="row"><li>drtext</li><li>Get data resource entries complete text</li></ul>
<ul class="row"><li>entret</li><li>Retrieve sequence entries from flatfile databases and files</li></ul>
<ul class="row"><li>ontotext</li><li>Get ontology term(s) original full text</li></ul>
<ul class="row"><li>textget</li><li>Get text data entries</li></ul>
<ul class="row"><li>textsearch</li><li>Search the textual description of sequence(s)</li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Tool data retrieval</li>
<ul class="group">
<ul class="row"><li>acdc</li><li>Test an application ACD file</li></ul>
<ul class="row"><li>acdpretty</li><li>Correctly reformat an application ACD file</li></ul>
<ul class="row"><li>acdtable</li><li>Generate an HTML table of parameters from an application ACD file</li></ul>
<ul class="row"><li>acdtrace</li><li>Trace processing of an application ACD file (for testing)</li></ul>
<ul class="row"><li>acdvalid</li><li>Validate an application ACD file</li></ul>
</ul>
<li class="group_title">Data retrieval &#8212; Url data retrieval</li>
<ul class="group">
<ul class="row"><li>urlget</li><li>Get URLs of data resources</li></ul>
</ul>
<li class="group_title">Display &#8212; Publication-quality display</li>
<ul class="group">
<ul class="row"><li>abiview</li><li>Display the trace in an ABI sequencer file</li></ul>
<ul class="row"><li>cirdna</li><li>Draw circular map of DNA constructs</li></ul>
<ul class="row"><li>iep</li><li>Calculate the isoelectric point of proteins</li></ul>
<ul class="row"><li>lindna</li><li>Draw linear maps of DNA constructs</li></ul>
<ul class="row"><li>pepinfo</li><li>Plot amino acid properties of a protein sequence in parallel</li></ul>
<ul class="row"><li>pepnet</li><li>Draw a helical net for a protein sequence</li></ul>
<ul class="row"><li>pepwheel</li><li>Draw a helical wheel diagram for a protein sequence</li></ul>
<ul class="row"><li>plotorf</li><li>Plot potential open reading frames in a nucleotide sequence</li></ul>
<ul class="row"><li>prettyplot</li><li>Draw a sequence alignment with pretty formatting</li></ul>
<ul class="row"><li>prettyseq</li><li>Write a nucleotide sequence and its translation to file</li></ul>
<ul class="row"><li>remap</li><li>Display restriction enzyme binding sites in a nucleotide sequence</li></ul>
<ul class="row"><li>showfeat</li><li>Display features of a sequence in pretty format</li></ul>
<ul class="row"><li>showpep</li><li>Display protein sequences with features in pretty format</li></ul>
<ul class="row"><li>sixpack</li><li>Display a DNA sequence with 6-frame translation and ORFs</li></ul>
<ul class="row"><li>topo</li><li>Draw an image of a transmembrane protein</li></ul>
</ul>
<li class="group_title">Documentation &#8212; Documentation</li>
<ul class="group">
<ul class="row"><li>seealso</li><li>Find programs with similar function to a specified program</li></ul>
<ul class="row"><li>tfm</li><li>Display full documentation for an application</li></ul>
<ul class="row"><li>wossdata</li><li>Find programs by EDAM data</li></ul>
<ul class="row"><li>wossinput</li><li>Find programs by EDAM input data</li></ul>
<ul class="row"><li>wossname</li><li>Find programs by keywords in their short description</li></ul>
<ul class="row"><li>wossoperation</li><li>Find programs by EDAM operation</li></ul>
<ul class="row"><li>wossoutput</li><li>Find programs by EDAM output data</li></ul>
<ul class="row"><li>wossparam</li><li>Find programs by EDAM parameter</li></ul>
<ul class="row"><li>wosstopic</li><li>Find programs by EDAM topic</li></ul>
</ul>
<li class="group_title">Edit &#8212; Data file and content editing</li>
<ul class="group">
<ul class="row"><li>aligncopy</li><li>Read and write alignments</li></ul>
<ul class="row"><li>aligncopypair</li><li>Read and write pairs from alignments</li></ul>
<ul class="row"><li>biosed</li><li>Replace or delete sequence sections</li></ul>
<ul class="row"><li>codcopy</li><li>Copy and reformat a codon usage table</li></ul>
<ul class="row"><li>cutseq</li><li>Remove a section from a sequence</li></ul>
<ul class="row"><li>degapseq</li><li>Remove non-alphabetic (e.g. gap) characters from sequences</li></ul>
<ul class="row"><li>descseq</li><li>Alter the name or description of a sequence</li></ul>
<ul class="row"><li>entret</li><li>Retrieve sequence entries from flatfile databases and files</li></ul>
<ul class="row"><li>extractalign</li><li>Extract regions from a sequence alignment</li></ul>
<ul class="row"><li>extractfeat</li><li>Extract features from sequence(s)</li></ul>
<ul class="row"><li>extractseq</li><li>Extract regions from a sequence</li></ul>
<ul class="row"><li>featcopy</li><li>Read and write a feature table</li></ul>
<ul class="row"><li>featmerge</li><li>Merge two overlapping feature tables</li></ul>
<ul class="row"><li>featreport</li><li>Read and write a feature table</li></ul>
<ul class="row"><li>feattext</li><li>Return a feature table original text</li></ul>
<ul class="row"><li>listor</li><li>Write a list file of the logical OR of two sets of sequences</li></ul>
<ul class="row"><li>makenucseq</li><li>Create random nucleotide sequences</li></ul>
<ul class="row"><li>makeprotseq</li><li>Create random protein sequences</li></ul>
<ul class="row"><li>maskambignuc</li><li>Mask all ambiguity characters in nucleotide sequences with N</li></ul>
<ul class="row"><li>maskambigprot</li><li>Mask all ambiguity characters in protein sequences with X</li></ul>
<ul class="row"><li>maskfeat</li><li>Write a sequence with masked features</li></ul>
<ul class="row"><li>maskseq</li><li>Write a sequence with masked regions</li></ul>
<ul class="row"><li>newseq</li><li>Create a sequence file from a typed-in sequence</li></ul>
<ul class="row"><li>nohtml</li><li>Remove mark-up (e.g. HTML tags) from an ASCII text file</li></ul>
<ul class="row"><li>noreturn</li><li>Remove carriage return from ASCII files</li></ul>
<ul class="row"><li>nospace</li><li>Remove whitespace from an ASCII text file</li></ul>
<ul class="row"><li>notab</li><li>Replace tabs with spaces in an ASCII text file</li></ul>
<ul class="row"><li>notseq</li><li>Write to file a subset of an input stream of sequences</li></ul>
<ul class="row"><li>nthseq</li><li>Write to file a single sequence from an input stream of sequences</li></ul>
<ul class="row"><li>nthseqset</li><li>Read and write (return) one set of sequences from many</li></ul>
<ul class="row"><li>pasteseq</li><li>Insert one sequence into another</li></ul>
<ul class="row"><li>revseq</li><li>Reverse and complement a nucleotide sequence</li></ul>
<ul class="row"><li>seqcount</li><li>Read and count sequences</li></ul>
<ul class="row"><li>seqret</li><li>Read and write (return) sequences</li></ul>
<ul class="row"><li>seqretsetall</li><li>Read and write (return) many sets of sequences</li></ul>
<ul class="row"><li>seqretsplit</li><li>Read sequences and write them to individual files</li></ul>
<ul class="row"><li>sizeseq</li><li>Sort sequences by size</li></ul>
<ul class="row"><li>skipredundant</li><li>Remove redundant sequences from an input set</li></ul>
<ul class="row"><li>skipseq</li><li>Read and write (return) sequences, skipping first few</li></ul>
<ul class="row"><li>splitsource</li><li>Split sequence(s) into original source sequences</li></ul>
<ul class="row"><li>splitter</li><li>Split sequence(s) into smaller sequences</li></ul>
<ul class="row"><li>trimest</li><li>Remove poly-A tails from nucleotide sequences</li></ul>
<ul class="row"><li>trimseq</li><li>Remove unwanted characters from start and end of sequence(s)</li></ul>
<ul class="row"><li>trimspace</li><li>Remove extra whitespace from an ASCII text file</li></ul>
<ul class="row"><li>union</li><li>Concatenate multiple sequences into a single sequence</li></ul>
<ul class="row"><li>vectorstrip</li><li>Remove vectors from the ends of nucleotide sequence(s)</li></ul>
<ul class="row"><li>yank</li><li>Add a sequence reference (a full USA) to a list file</li></ul>
<ul class="row"><li>seqfraggle</li><li>Remove fragment sequences from DHF files</li></ul>
</ul>
<li class="group_title">Enzyme kinetics &#8212; Enzyme kinetics calculations</li>
<ul class="group">
<ul class="row"><li>findkm</li><li>Calculate and plot enzyme reaction data</li></ul>
</ul>
<li class="group_title">Feature tables &#8212; Manipulation and display of sequence annotation</li>
<ul class="group">
<ul class="row"><li>extractfeat</li><li>Extract features from sequence(s)</li></ul>
<ul class="row"><li>maskfeat</li><li>Write a sequence with masked features</li></ul>
<ul class="row"><li>showfeat</li><li>Display features of a sequence in pretty format</li></ul>
<ul class="row"><li>twofeat</li><li>Find neighbouring pairs of features in sequence(s)</li></ul>
</ul>
<li class="group_title">HMM &#8212; Hidden markov model analysis</li>
<ul class="group">
<ul class="row"><li>ohmmalign</li><li>Align sequences with an HMM</li></ul>
<ul class="row"><li>ohmmbuild</li><li>Build HMM</li></ul>
<ul class="row"><li>ohmmcalibrate</li><li>Calibrate a hidden Markov model</li></ul>
<ul class="row"><li>ohmmconvert</li><li>Convert between HMM formats</li></ul>
<ul class="row"><li>ohmmemit</li><li>Extract HMM sequences</li></ul>
<ul class="row"><li>ohmmfetch</li><li>Extract HMM from a database</li></ul>
<ul class="row"><li>ohmmindex</li><li>Index an HMM database</li></ul>
<ul class="row"><li>ohmmpfam</li><li>Align single sequence with an HMM</li></ul>
<ul class="row"><li>ohmmsearch</li><li>Search sequence database with an HMM</li></ul>
<ul class="row"><li>ehmmalign</li><li>Align sequences to an HMM profile</li></ul>
<ul class="row"><li>ehmmbuild</li><li>Build a profile HMM from an alignment</li></ul>
<ul class="row"><li>ehmmcalibrate</li><li>Calibrate HMM search statistics</li></ul>
<ul class="row"><li>ehmmconvert</li><li>Convert between profile HMM file formats</li></ul>
<ul class="row"><li>ehmmemit</li><li>Generate sequences from a profile HMM</li></ul>
<ul class="row"><li>ehmmfetch</li><li>Retrieve an HMM from an HMM database</li></ul>
<ul class="row"><li>ehmmindex</li><li>Create a binary SSI index for an HMM database</li></ul>
<ul class="row"><li>ehmmpfam</li><li>Search one or more sequences against an HMM database</li></ul>
<ul class="row"><li>ehmmsearch</li><li>Search a sequence database with a profile HMM</li></ul>
<ul class="row"><li>libgen</li><li>Generate discriminating elements from alignments</li></ul>
</ul>
<li class="group_title">Menus &#8212; Menu interface(s)</li>
<ul class="group">
<ul class="row"><li>emnu</li><li>Simple menu of EMBOSS applications</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid secondary structure</li>
<ul class="group">
<ul class="row"><li>banana</li><li>Plot bending and curvature data for B-DNA</li></ul>
<ul class="row"><li>btwisted</li><li>Calculate the twisting in a B-DNA sequence</li></ul>
<ul class="row"><li>einverted</li><li>Find inverted repeats in nucleotide sequences</li></ul>
<ul class="row"><li>sirna</li><li>Find siRNA duplexes in mRNA</li></ul>
<ul class="row"><li>ovrnaalifold</li><li>Calculate secondary structures for a set of aligned RNAs</li></ul>
<ul class="row"><li>ovrnaalifoldpf</li><li>Calculate secondary structures for a set of aligned RNAs (partition)</li></ul>
<ul class="row"><li>ovrnacofold</li><li>Calculate secondary structures of RNA dimers</li></ul>
<ul class="row"><li>ovrnacofoldconc</li><li>Calculate secondary structures of RNA dimers (concentrations)</li></ul>
<ul class="row"><li>ovrnacofoldpf</li><li>Calculate secondary structures of RNA dimers (partitioning)</li></ul>
<ul class="row"><li>ovrnadistance</li><li>Calculate distances between RNA secondary structures</li></ul>
<ul class="row"><li>ovrnaduplex</li><li>Predict RNA duplex (hybridization) sites and structure</li></ul>
<ul class="row"><li>ovrnaeval</li><li>Calculate energy of RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>ovrnaevalpair</li><li>Calculate energy of RNA sequences on given secondary structure</li></ul>
<ul class="row"><li>ovrnafold</li><li>Calculate min. energy RNA structure / pair probabilities (partition)</li></ul>
<ul class="row"><li>ovrnafoldpf</li><li>Calculate min. energy RNA structure / pair probabilities</li></ul>
<ul class="row"><li>ovrnaheat</li><li>Calculate specific heat of RNA melting</li></ul>
<ul class="row"><li>ovrnainverse</li><li>Find RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>ovrnalfold</li><li>Calculate locally stable secondary structures of RNAs</li></ul>
<ul class="row"><li>ovrnaplot</li><li>Draw RNA secondary structures</li></ul>
<ul class="row"><li>ovrnasubopt</li><li>Calculate suboptimal secondary structure of RNA</li></ul>
<ul class="row"><li>vrna2dfold</li><li>Calculate RNA structures and samples of k,l neighbourhoods</li></ul>
<ul class="row"><li>vrnaaliduplex</li><li>RNA duplex calculation for two sequence alignments</li></ul>
<ul class="row"><li>vrnaalifold</li><li>Calculate secondary structures for a set of aligned RNAs</li></ul>
<ul class="row"><li>vrnaalifoldpf</li><li>Calculate secondary structures for a set of aligned RNAs (partition)</li></ul>
<ul class="row"><li>vrnacofold</li><li>Calculate secondary structures of RNA dimers</li></ul>
<ul class="row"><li>vrnacofoldconc</li><li>Calculate secondary structures of RNA dimers (concentrations)</li></ul>
<ul class="row"><li>vrnacofoldpf</li><li>Calculate secondary structures of RNA dimers (partitioning)</li></ul>
<ul class="row"><li>vrnadistance</li><li>Calculate distances between RNA secondary structures</li></ul>
<ul class="row"><li>vrnaduplex</li><li>Predict RNA duplex (hybridization) sites and structure</li></ul>
<ul class="row"><li>vrnaeval</li><li>Calculate energy of RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>vrnaevalpair</li><li>Calculate energy of RNA sequences on given secondary structure</li></ul>
<ul class="row"><li>vrnafold</li><li>Calculate min. energy RNA secondary structures and pair probabilities</li></ul>
<ul class="row"><li>vrnafoldpf</li><li>Calculate min. energy RNA structures / pair probabilities (partition)</li></ul>
<ul class="row"><li>vrnaheat</li><li>Calculate specific heat of RNA melting</li></ul>
<ul class="row"><li>vrnainverse</li><li>Find RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>vrnalalifoldpf</li><li>Calculate secondary structures for a set of aligned RNAs (partition)</li></ul>
<ul class="row"><li>vrnalfold</li><li>Calculate locally stable secondary structures of RNAs</li></ul>
<ul class="row"><li>vrnalfoldz</li><li>Calculate locally stable secondary structures of RNAs plus zscore</li></ul>
<ul class="row"><li>vrnapkplex</li><li>Calculate RNA structures plus pseudoknots</li></ul>
<ul class="row"><li>vrnaplfold</li><li>Compute avg. pair probabilities for local base pairs in RNA sequences</li></ul>
<ul class="row"><li>vrnaplot</li><li>Draw RNA secondary structures</li></ul>
<ul class="row"><li>vrnasubopt</li><li>Calculate suboptimal secondary structures of RNAs</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Codon usage analysis</li>
<ul class="group">
<ul class="row"><li>cai</li><li>Calculate codon adaptation index</li></ul>
<ul class="row"><li>chips</li><li>Calculate Nc codon usage statistic</li></ul>
<ul class="row"><li>codcmp</li><li>Codon usage table comparison</li></ul>
<ul class="row"><li>codcopy</li><li>Copy and reformat a codon usage table</li></ul>
<ul class="row"><li>cusp</li><li>Create a codon usage table from nucleotide sequence(s)</li></ul>
<ul class="row"><li>syco</li><li>Draw synonymous codon usage statistic plot for a nucleotide sequence</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Composition of nucleotide sequences</li>
<ul class="group">
<ul class="row"><li>banana</li><li>Plot bending and curvature data for B-DNA</li></ul>
<ul class="row"><li>btwisted</li><li>Calculate the twisting in a B-DNA sequence</li></ul>
<ul class="row"><li>chaos</li><li>Draw a chaos game representation plot for a nucleotide sequence</li></ul>
<ul class="row"><li>compseq</li><li>Calculate the composition of unique words in sequences</li></ul>
<ul class="row"><li>dan</li><li>Calculate nucleic acid melting temperature</li></ul>
<ul class="row"><li>density</li><li>Draw a nucleic acid density plot</li></ul>
<ul class="row"><li>freak</li><li>Generate residue/base frequency table or plot</li></ul>
<ul class="row"><li>isochore</li><li>Plot isochores in DNA sequences</li></ul>
<ul class="row"><li>wordcount</li><li>Count and extract unique words in molecular sequence(s)</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; CpG island detection and analysis</li>
<ul class="group">
<ul class="row"><li>cpgplot</li><li>Identify and plot CpG islands in nucleotide sequence(s)</li></ul>
<ul class="row"><li>cpgreport</li><li>Identify and report CpG-rich regions in nucleotide sequence(s)</li></ul>
<ul class="row"><li>geecee</li><li>Calculate fractional GC content of nucleic acid sequences</li></ul>
<ul class="row"><li>newcpgreport</li><li>Identify CpG islands in nucleotide sequence(s)</li></ul>
<ul class="row"><li>newcpgseek</li><li>Identify and report CpG-rich regions in nucleotide sequence(s)</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid functional sites</li>
<ul class="group">
<ul class="row"><li>marscan</li><li>Find matrix/scaffold recognition (MRS) signatures in DNA sequences</li></ul>
<ul class="row"><li>sirna</li><li>Find siRNA duplexes in mRNA</li></ul>
<ul class="row"><li>trimest</li><li>Remove poly-A tails from nucleotide sequences</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Predictions of genes and other genomic features</li>
<ul class="group">
<ul class="row"><li>checktrans</li><li>Report STOP codons and ORF statistics of a protein</li></ul>
<ul class="row"><li>getorf</li><li>Find and extract open reading frames (ORFs)</li></ul>
<ul class="row"><li>marscan</li><li>Find matrix/scaffold recognition (MRS) signatures in DNA sequences</li></ul>
<ul class="row"><li>plotorf</li><li>Plot potential open reading frames in a nucleotide sequence</li></ul>
<ul class="row"><li>showorf</li><li>Display a nucleotide sequence and translation in pretty format</li></ul>
<ul class="row"><li>sixpack</li><li>Display a DNA sequence with 6-frame translation and ORFs</li></ul>
<ul class="row"><li>syco</li><li>Draw synonymous codon usage statistic plot for a nucleotide sequence</li></ul>
<ul class="row"><li>tcode</li><li>Identify protein-coding regions using Fickett TESTCODE statistic</li></ul>
<ul class="row"><li>wobble</li><li>Plot third base position variability in a nucleotide sequence</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid motif searches</li>
<ul class="group">
<ul class="row"><li>dreg</li><li>Regular expression search of nucleotide sequence(s)</li></ul>
<ul class="row"><li>fuzznuc</li><li>Search for patterns in nucleotide sequences</li></ul>
<ul class="row"><li>fuzztran</li><li>Search for patterns in protein sequences (translated)</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid sequence mutation</li>
<ul class="group">
<ul class="row"><li>msbar</li><li>Mutate a sequence</li></ul>
<ul class="row"><li>shuffleseq</li><li>Shuffle a set of sequences maintaining composition</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Primer prediction</li>
<ul class="group">
<ul class="row"><li>eprimer3</li><li>Pick PCR primers and hybridization oligos</li></ul>
<ul class="row"><li>eprimer32</li><li>Pick PCR primers and hybridization oligos</li></ul>
<ul class="row"><li>primersearch</li><li>Search DNA sequences for matches with primer pairs</li></ul>
<ul class="row"><li>stssearch</li><li>Search a DNA database for matches with a set of STS primers</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid profile generation and searching</li>
<ul class="group">
<ul class="row"><li>profit</li><li>Scan one or more sequences with a simple frequency matrix</li></ul>
<ul class="row"><li>prophecy</li><li>Create frequency matrix or profile from a multiple alignment</li></ul>
<ul class="row"><li>prophet</li><li>Scan one or more sequences with a Gribskov or Henikoff profile</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid physicochemical properties</li>
<ul class="group">
<ul class="row"><li>banana</li><li>Plot bending and curvature data for B-DNA</li></ul>
<ul class="row"><li>btwisted</li><li>Calculate the twisting in a B-DNA sequence</li></ul>
<ul class="row"><li>dan</li><li>Calculate nucleic acid melting temperature</li></ul>
<ul class="row"><li>vrnaheat</li><li>Calculate specific heat of RNA melting</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Nucleic acid repeat detection</li>
<ul class="group">
<ul class="row"><li>einverted</li><li>Find inverted repeats in nucleotide sequences</li></ul>
<ul class="row"><li>equicktandem</li><li>Find tandem repeats in nucleotide sequences</li></ul>
<ul class="row"><li>etandem</li><li>Find tandem repeats in a nucleotide sequence</li></ul>
<ul class="row"><li>palindrome</li><li>Find inverted repeats in nucleotide sequence(s)</li></ul>
</ul>
<li class="group_title">Nucleic &#8212; Restriction enzyme sites in nucleotide sequences</li>
<ul class="group">
<ul class="row"><li>recoder</li><li>Find restriction sites to remove (mutate) with no translation change</li></ul>
<ul class="row"><li>redata</li><li>Retrieve information from REBASE restriction enzyme database</li></ul>
<ul class="row"><li>remap</li><li>Display restriction enzyme binding sites in a nucleotide sequence</li></ul>
<ul class="row"><li>restover</li><li>Find restriction enzymes producing a specific overhang</li></ul>
<ul class="row"><li>restrict</li><li>Report restriction enzyme cleavage sites in a nucleotide sequence</li></ul>
<ul class="row"><li>showseq</li><li>Display sequences with features in pretty format</li></ul>
<ul class="row"><li>silent</li><li>Find restriction sites to insert (mutate) with no translation change</li></ul>
</ul>
<li class="group_title">Nucleic RNA folding &#8212; RNA folding methods and analysis</li>
<ul class="group">
<ul class="row"><li>ovrnaalifold</li><li>Calculate secondary structures for a set of aligned RNAs</li></ul>
<ul class="row"><li>ovrnaalifoldpf</li><li>Calculate secondary structures for a set of aligned RNAs (partition)</li></ul>
<ul class="row"><li>ovrnacofold</li><li>Calculate secondary structures of RNA dimers</li></ul>
<ul class="row"><li>ovrnacofoldconc</li><li>Calculate secondary structures of RNA dimers (concentrations)</li></ul>
<ul class="row"><li>ovrnacofoldpf</li><li>Calculate secondary structures of RNA dimers (partitioning)</li></ul>
<ul class="row"><li>ovrnadistance</li><li>Calculate distances between RNA secondary structures</li></ul>
<ul class="row"><li>ovrnaduplex</li><li>Predict RNA duplex (hybridization) sites and structure</li></ul>
<ul class="row"><li>ovrnaeval</li><li>Calculate energy of RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>ovrnaevalpair</li><li>Calculate energy of RNA sequences on given secondary structure</li></ul>
<ul class="row"><li>ovrnafold</li><li>Calculate min. energy RNA structure / pair probabilities (partition)</li></ul>
<ul class="row"><li>ovrnafoldpf</li><li>Calculate min. energy RNA structure / pair probabilities</li></ul>
<ul class="row"><li>ovrnaheat</li><li>Calculate specific heat of RNA melting</li></ul>
<ul class="row"><li>ovrnainverse</li><li>Find RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>ovrnalfold</li><li>Calculate locally stable secondary structures of RNAs</li></ul>
<ul class="row"><li>ovrnaplot</li><li>Draw RNA secondary structures</li></ul>
<ul class="row"><li>ovrnasubopt</li><li>Calculate suboptimal secondary structure of RNA</li></ul>
<ul class="row"><li>vrna2dfold</li><li>Calculate RNA structures and samples of k,l neighbourhoods</li></ul>
<ul class="row"><li>vrnaaliduplex</li><li>RNA duplex calculation for two sequence alignments</li></ul>
<ul class="row"><li>vrnaalifold</li><li>Calculate secondary structures for a set of aligned RNAs</li></ul>
<ul class="row"><li>vrnaalifoldpf</li><li>Calculate secondary structures for a set of aligned RNAs (partition)</li></ul>
<ul class="row"><li>vrnacofold</li><li>Calculate secondary structures of RNA dimers</li></ul>
<ul class="row"><li>vrnacofoldconc</li><li>Calculate secondary structures of RNA dimers (concentrations)</li></ul>
<ul class="row"><li>vrnacofoldpf</li><li>Calculate secondary structures of RNA dimers (partitioning)</li></ul>
<ul class="row"><li>vrnadistance</li><li>Calculate distances between RNA secondary structures</li></ul>
<ul class="row"><li>vrnaduplex</li><li>Predict RNA duplex (hybridization) sites and structure</li></ul>
<ul class="row"><li>vrnaeval</li><li>Calculate energy of RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>vrnaevalpair</li><li>Calculate energy of RNA sequences on given secondary structure</li></ul>
<ul class="row"><li>vrnafold</li><li>Calculate min. energy RNA secondary structures and pair probabilities</li></ul>
<ul class="row"><li>vrnafoldpf</li><li>Calculate min. energy RNA structures / pair probabilities (partition)</li></ul>
<ul class="row"><li>vrnaheat</li><li>Calculate specific heat of RNA melting</li></ul>
<ul class="row"><li>vrnainverse</li><li>Find RNA sequences with a given secondary structure</li></ul>
<ul class="row"><li>vrnalalifoldpf</li><li>Calculate secondary structures for a set of aligned RNAs (partition)</li></ul>
<ul class="row"><li>vrnalfold</li><li>Calculate locally stable secondary structures of RNAs</li></ul>
<ul class="row"><li>vrnalfoldz</li><li>Calculate locally stable secondary structures of RNAs plus zscore</li></ul>
<ul class="row"><li>vrnapkplex</li><li>Calculate RNA structures plus pseudoknots</li></ul>
<ul class="row"><li>vrnaplfold</li><li>Compute avg. pair probabilities for local base pairs in RNA sequences</li></ul>
<ul class="row"><li>vrnaplot</li><li>Draw RNA secondary structures</li></ul>
<ul class="row"><li>vrnasubopt</li><li>Calculate suboptimal secondary structures of RNAs</li></ul>
</ul>
<li class="group_title">Nucleic transcription &#8212;Transcription factors, promoters and terminator prediction</li>
<ul class="group">
<ul class="row"><li>jaspscan</li><li>Scan DNA sequences for transcription factors</li></ul>
<ul class="row"><li>marscan</li><li>Find matrix/scaffold recognition (MRS) signatures in DNA sequences</li></ul>
<ul class="row"><li>tfscan</li><li>Identify transcription factor binding sites in DNA sequences</li></ul>
</ul>
<li class="group_title">Nucleic translation &#8212;Translation of nucleotide sequence to protein sequence</li>
<ul class="group">
<ul class="row"><li>backtranambig</li><li>Back-translate a protein sequence to ambiguous nucleotide sequence</li></ul>
<ul class="row"><li>backtranseq</li><li>Back-translate a protein sequence to a nucleotide sequence</li></ul>
<ul class="row"><li>checktrans</li><li>Report STOP codons and ORF statistics of a protein</li></ul>
<ul class="row"><li>coderet</li><li>Extract CDS, mRNA and translations from feature tables</li></ul>
<ul class="row"><li>plotorf</li><li>Plot potential open reading frames in a nucleotide sequence</li></ul>
<ul class="row"><li>prettyseq</li><li>Write a nucleotide sequence and its translation to file</li></ul>
<ul class="row"><li>remap</li><li>Display restriction enzyme binding sites in a nucleotide sequence</li></ul>
<ul class="row"><li>showorf</li><li>Display a nucleotide sequence and translation in pretty format</li></ul>
<ul class="row"><li>showseq</li><li>Display sequences with features in pretty format</li></ul>
<ul class="row"><li>sixpack</li><li>Display a DNA sequence with 6-frame translation and ORFs</li></ul>
<ul class="row"><li>transeq</li><li>Translate nucleic acid sequences</li></ul>
</ul>
<li class="group_title">Ontology &#8212; Ontologies</li>
<ul class="group">
<ul class="row"><li>ontocount</li><li>Count ontology term(s)</li></ul>
<ul class="row"><li>ontoget</li><li>Get ontology term(s)</li></ul>
<ul class="row"><li>ontogetcommon</li><li>Get common ancestor for terms</li></ul>
<ul class="row"><li>ontogetdown</li><li>Get ontology term(s) by parent id</li></ul>
<ul class="row"><li>ontogetobsolete</li><li>Get ontology ontology terms</li></ul>
<ul class="row"><li>ontogetroot</li><li>Get ontology root terms by child identifier</li></ul>
<ul class="row"><li>ontogetsibs</li><li>Get ontology term(s) by id with common parent</li></ul>
<ul class="row"><li>ontogetup</li><li>Get ontology term(s) by id of child</li></ul>
<ul class="row"><li>ontoisobsolete</li><li>Report whether an ontology term id is obsolete</li></ul>
<ul class="row"><li>ontotext</li><li>Get ontology term(s) original full text</li></ul>
</ul>
<li class="group_title">Ontology edam &#8212; Edam ontology</li>
<ul class="group">
<ul class="row"><li>drfinddata</li><li>Find public databases by data type</li></ul>
<ul class="row"><li>drfindformat</li><li>Find public databases by format</li></ul>
<ul class="row"><li>drfindid</li><li>Find public databases by identifier</li></ul>
<ul class="row"><li>drfindresource</li><li>Find public databases by resource</li></ul>
<ul class="row"><li>edamdef</li><li>Find EDAM ontology terms by definition</li></ul>
<ul class="row"><li>edamhasinput</li><li>Find EDAM ontology terms by has_input relation</li></ul>
<ul class="row"><li>edamhasoutput</li><li>Find EDAM ontology terms by has_output relation</li></ul>
<ul class="row"><li>edamisformat</li><li>Find EDAM ontology terms by is_format_of relation</li></ul>
<ul class="row"><li>edamisid</li><li>Find EDAM ontology terms by is_identifier_of relation</li></ul>
<ul class="row"><li>edamname</li><li>Find EDAM ontology terms by name</li></ul>
<ul class="row"><li>wossdata</li><li>Find programs by EDAM data</li></ul>
<ul class="row"><li>wossinput</li><li>Find programs by EDAM input data</li></ul>
<ul class="row"><li>wossoperation</li><li>Find programs by EDAM operation</li></ul>
<ul class="row"><li>wossoutput</li><li>Find programs by EDAM output data</li></ul>
<ul class="row"><li>wossparam</li><li>Find programs by EDAM parameter</li></ul>
<ul class="row"><li>wosstopic</li><li>Find programs by EDAM topic</li></ul>
</ul>
<li class="group_title">Ontology go &#8212; Go gene ontology</li>
<ul class="group">
<ul class="row"><li>godef</li><li>Find GO ontology terms by definition</li></ul>
<ul class="row"><li>goname</li><li>Find GO ontology terms by name</li></ul>
</ul>
<li class="group_title">Ontology taxonomy &#8212; Ncbi taxonomy</li>
<ul class="group">
<ul class="row"><li>taxget</li><li>Get taxon(s)</li></ul>
<ul class="row"><li>taxgetdown</li><li>Get descendants of taxon(s)</li></ul>
<ul class="row"><li>taxgetrank</li><li>Get parents of taxon(s)</li></ul>
<ul class="row"><li>taxgetspecies</li><li>Get all species under taxon(s)</li></ul>
<ul class="row"><li>taxgetup</li><li>Get parents of taxon(s)</li></ul>
</ul>
<li class="group_title">Phylogeny consensus &#8212; Phylogenetic consensus methods</li>
<ul class="group">
<ul class="row"><li>econsense</li><li>Majority-rule and strict consensus tree</li></ul>
<ul class="row"><li>fconsense</li><li>Majority-rule and strict consensus tree</li></ul>
<ul class="row"><li>ftreedist</li><li>Calculate distances between trees</li></ul>
<ul class="row"><li>ftreedistpair</li><li>Calculate distance between two sets of trees</li></ul>
</ul>
<li class="group_title">Phylogeny &#8212; Phylogenetic continuous character methods</li>
<ul class="group">
<ul class="row"><li>econtml</li><li>Continuous character maximum likelihood method</li></ul>
<ul class="row"><li>econtrast</li><li>Continuous character contrasts</li></ul>
<ul class="row"><li>fcontrast</li><li>Continuous character contrasts</li></ul>
</ul>
<li class="group_title">Phylogeny &#8212; Phylogenetic discrete character methods</li>
<ul class="group">
<ul class="row"><li>eclique</li><li>Largest clique program</li></ul>
<ul class="row"><li>edollop</li><li>Dollo and polymorphism parsimony algorithm</li></ul>
<ul class="row"><li>edolpenny</li><li>Penny algorithm Dollo or polymorphism</li></ul>
<ul class="row"><li>efactor</li><li>Multistate to binary recoding program</li></ul>
<ul class="row"><li>emix</li><li>Mixed parsimony algorithm</li></ul>
<ul class="row"><li>epenny</li><li>Penny algorithm, branch-and-bound</li></ul>
<ul class="row"><li>fclique</li><li>Largest clique program</li></ul>
<ul class="row"><li>fdollop</li><li>Dollo and polymorphism parsimony algorithm</li></ul>
<ul class="row"><li>fdolpenny</li><li>Penny algorithm Dollo or polymorphism</li></ul>
<ul class="row"><li>ffactor</li><li>Multistate to binary recoding program</li></ul>
<ul class="row"><li>fmix</li><li>Mixed parsimony algorithm</li></ul>
<ul class="row"><li>fmove</li><li>Interactive mixed method parsimony</li></ul>
<ul class="row"><li>fpars</li><li>Discrete character parsimony</li></ul>
<ul class="row"><li>fpenny</li><li>Penny algorithm, branch-and-bound</li></ul>
</ul>
<li class="group_title">Phylogeny &#8212; Phylogenetic distance matrix methods</li>
<ul class="group">
<ul class="row"><li>efitch</li><li>Fitch-Margoliash and least-squares distance methods</li></ul>
<ul class="row"><li>ekitsch</li><li>Fitch-Margoliash method with contemporary tips</li></ul>
<ul class="row"><li>eneighbor</li><li>Phylogenies from distance matrix by N-J or UPGMA method</li></ul>
<ul class="row"><li>ffitch</li><li>Fitch-Margoliash and least-squares distance methods</li></ul>
<ul class="row"><li>fkitsch</li><li>Fitch-Margoliash method with contemporary tips</li></ul>
<ul class="row"><li>fneighbor</li><li>Phylogenies from distance matrix by N-J or UPGMA method</li></ul>
</ul>
<li class="group_title">Phylogeny &#8212; Phylogenetic gene frequency methods</li>
<ul class="group">
<ul class="row"><li>egendist</li><li>Genetic distance matrix program</li></ul>
<ul class="row"><li>fcontml</li><li>Gene frequency and continuous character maximum likelihood</li></ul>
<ul class="row"><li>fgendist</li><li>Compute genetic distances from gene frequencies</li></ul>
</ul>
<li class="group_title">Phylogeny &#8212; Phylogenetic molecular sequence methods</li>
<ul class="group">
<ul class="row"><li>distmat</li><li>Create a distance matrix from a multiple sequence alignment</li></ul>
<ul class="row"><li>ednacomp</li><li>DNA compatibility algorithm</li></ul>
<ul class="row"><li>ednadist</li><li>Nucleic acid sequence distance matrix program</li></ul>
<ul class="row"><li>ednainvar</li><li>Nucleic acid sequence invariants method</li></ul>
<ul class="row"><li>ednaml</li><li>Phylogenies from nucleic acid maximum likelihood</li></ul>
<ul class="row"><li>ednamlk</li><li>Phylogenies from nucleic acid maximum likelihood with clock</li></ul>
<ul class="row"><li>ednapars</li><li>DNA parsimony algorithm</li></ul>
<ul class="row"><li>ednapenny</li><li>Penny algorithm for DNA</li></ul>
<ul class="row"><li>eprotdist</li><li>Protein distance algorithm</li></ul>
<ul class="row"><li>eprotpars</li><li>Protein parsimony algorithm</li></ul>
<ul class="row"><li>erestml</li><li>Restriction site maximum likelihood method</li></ul>
<ul class="row"><li>eseqboot</li><li>Bootstrapped sequences algorithm</li></ul>
<ul class="row"><li>fdiscboot</li><li>Bootstrapped discrete sites algorithm</li></ul>
<ul class="row"><li>fdnacomp</li><li>DNA compatibility algorithm</li></ul>
<ul class="row"><li>fdnadist</li><li>Nucleic acid sequence distance matrix program</li></ul>
<ul class="row"><li>fdnainvar</li><li>Nucleic acid sequence invariants method</li></ul>
<ul class="row"><li>fdnaml</li><li>Estimate nucleotide phylogeny by maximum likelihood</li></ul>
<ul class="row"><li>fdnamlk</li><li>Estimates nucleotide phylogeny by maximum likelihood</li></ul>
<ul class="row"><li>fdnamove</li><li>Interactive DNA parsimony</li></ul>
<ul class="row"><li>fdnapars</li><li>DNA parsimony algorithm</li></ul>
<ul class="row"><li>fdnapenny</li><li>Penny algorithm for DNA</li></ul>
<ul class="row"><li>fdolmove</li><li>Interactive Dollo or polymorphism parsimony</li></ul>
<ul class="row"><li>ffreqboot</li><li>Bootstrapped genetic frequencies algorithm</li></ul>
<ul class="row"><li>fproml</li><li>Protein phylogeny by maximum likelihood</li></ul>
<ul class="row"><li>fpromlk</li><li>Protein phylogeny by maximum likelihood</li></ul>
<ul class="row"><li>fprotdist</li><li>Protein distance algorithm</li></ul>
<ul class="row"><li>fprotpars</li><li>Protein parsimony algorithm</li></ul>
<ul class="row"><li>frestboot</li><li>Bootstrapped restriction sites algorithm</li></ul>
<ul class="row"><li>frestdist</li><li>Calculate distance matrix from restriction sites or fragments</li></ul>
<ul class="row"><li>frestml</li><li>Restriction site maximum likelihood method</li></ul>
<ul class="row"><li>fseqboot</li><li>Bootstrapped sequences algorithm</li></ul>
<ul class="row"><li>fseqbootall</li><li>Bootstrapped sequences algorithm</li></ul>
</ul>
<li class="group_title">Phylogeny &#8212; Phylogenetic tree drawing methods</li>
<ul class="group">
<ul class="row"><li>fdrawgram</li><li>Plots a cladogram- or phenogram-like rooted tree diagram</li></ul>
<ul class="row"><li>fdrawtree</li><li>Plots an unrooted tree diagram</li></ul>
<ul class="row"><li>fretree</li><li>Interactive tree rearrangement</li></ul>
</ul>
<li class="group_title">Protein 2d structure &#8212; Protein secondary structure</li>
<ul class="group">
<ul class="row"><li>garnier</li><li>Predict protein secondary structure using GOR method</li></ul>
<ul class="row"><li>helixturnhelix</li><li>Identify nucleic acid-binding motifs in protein sequences</li></ul>
<ul class="row"><li>pepcoil</li><li>Predict coiled coil regions in protein sequences</li></ul>
<ul class="row"><li>pepnet</li><li>Draw a helical net for a protein sequence</li></ul>
<ul class="row"><li>pepwheel</li><li>Draw a helical wheel diagram for a protein sequence</li></ul>
</ul>
<li class="group_title">Protein 3d structure &#8212; Protein tertiary structure</li>
<ul class="group">
<ul class="row"><li>psiphi</li><li>Calculates phi and psi torsion angles from protein coordinates</li></ul>
<ul class="row"><li>domainreso</li><li>Remove low resolution domains from a DCF file</li></ul>
<ul class="row"><li>domainrep</li><li>Reorder DCF file to identify representative structures</li></ul>
<ul class="row"><li>contacts</li><li>Generate intra-chain CON files from CCF files</li></ul>
<ul class="row"><li>interface</li><li>Generate inter-chain CON files from CCF files</li></ul>
</ul>
<li class="group_title">Protein composition &#8212; Composition of protein sequences</li>
<ul class="group">
<ul class="row"><li>backtranambig</li><li>Back-translate a protein sequence to ambiguous nucleotide sequence</li></ul>
<ul class="row"><li>backtranseq</li><li>Back-translate a protein sequence to a nucleotide sequence</li></ul>
<ul class="row"><li>compseq</li><li>Calculate the composition of unique words in sequences</li></ul>
<ul class="row"><li>emowse</li><li>Search protein sequences by digest fragment molecular weight</li></ul>
<ul class="row"><li>freak</li><li>Generate residue/base frequency table or plot</li></ul>
<ul class="row"><li>mwcontam</li><li>Find weights common to multiple molecular weights files</li></ul>
<ul class="row"><li>mwfilter</li><li>Filter noisy data from molecular weights file</li></ul>
<ul class="row"><li>oddcomp</li><li>Identify proteins with specified sequence word composition</li></ul>
<ul class="row"><li>pepdigest</li><li>Report on protein proteolytic enzyme or reagent cleavage sites</li></ul>
<ul class="row"><li>pepinfo</li><li>Plot amino acid properties of a protein sequence in parallel</li></ul>
<ul class="row"><li>pepstats</li><li>Calculate statistics of protein properties</li></ul>
<ul class="row"><li>wordcount</li><li>Count and extract unique words in molecular sequence(s)</li></ul>
</ul>
<li class="group_title">Protein domains &#8212; Protein domain analysis</li>
<ul class="group">
<ul class="row"><li>helixturnhelix</li><li>Identify nucleic acid-binding motifs in protein sequences</li></ul>
<ul class="row"><li>pepcoil</li><li>Predict coiled coil regions in protein sequences</li></ul>
<ul class="row"><li>cathparse</li><li>Generate DCF file from raw CATH files</li></ul>
<ul class="row"><li>domainnr</li><li>Remove redundant domains from a DCF file</li></ul>
<ul class="row"><li>domainseqs</li><li>Add sequence records to a DCF file</li></ul>
<ul class="row"><li>domainsse</li><li>Add secondary structure records to a DCF file</li></ul>
<ul class="row"><li>scopparse</li><li>Generate DCF file from raw SCOP files</li></ul>
<ul class="row"><li>ssematch</li><li>Search a DCF file for secondary structure matches</li></ul>
<ul class="row"><li>domainalign</li><li>Generate alignments (DAF file) for nodes in a DCF file</li></ul>
<ul class="row"><li>domainrep</li><li>Reorder DCF file to identify representative structures</li></ul>
<ul class="row"><li>seqalign</li><li>Extend alignments (DAF file) with sequences (DHF file)</li></ul>
<ul class="row"><li>seqfraggle</li><li>Remove fragment sequences from DHF files</li></ul>
<ul class="row"><li>seqsort</li><li>Remove ambiguous classified sequences from DHF files</li></ul>
<ul class="row"><li>seqwords</li><li>Generate DHF files from keyword search of UniProt</li></ul>
<ul class="row"><li>libgen</li><li>Generate discriminating elements from alignments</li></ul>
<ul class="row"><li>matgen3d</li><li>Generate a 3D-1D scoring matrix from CCF files</li></ul>
<ul class="row"><li>rocon</li><li>Generate a hits file from comparing two DHF files</li></ul>
<ul class="row"><li>rocplot</li><li>Perform ROC analysis on hits files</li></ul>
</ul>
<li class="group_title">Protein functional sites &#8212; Protein functional sites</li>
<ul class="group">
<ul class="row"><li>sigcleave</li><li>Report on signal cleavage sites in a protein sequence</li></ul>
<ul class="row"><li>tmap</li><li>Predict and plot transmembrane segments in protein sequences</li></ul>
<ul class="row"><li>echlorop</li><li>Report presence of chloroplast transit peptides</li></ul>
<ul class="row"><li>elipop</li><li>Predict lipoproteins</li></ul>
<ul class="row"><li>esignalp</li><li>Report protein signal cleavage sites</li></ul>
<ul class="row"><li>etmhmm</li><li>Reports transmembrane helices</li></ul>
<ul class="row"><li>siggenlig</li><li>Generate ligand-binding signatures from a CON file</li></ul>
<ul class="row"><li>sigscan</li><li>Generate hits (DHF file) from a signature search</li></ul>
<ul class="row"><li>topo</li><li>Draw an image of a transmembrane protein</li></ul>
</ul>
<li class="group_title">Protein modifications &#8212; Protein post-translational modifications</li>
<ul class="group">
<ul class="row"><li>enetnglyc</li><li>Report N-glycosylation sites in human proteins</li></ul>
<ul class="row"><li>enetoglyc</li><li>Reports mucin type GalNAc O-glycosylation sites in mammalian proteins</li></ul>
<ul class="row"><li>enetphos</li><li>Report Ser, Thr and Tyr phosphorylation sites in eukaryotic proteins</li></ul>
<ul class="row"><li>eprop</li><li>Report propeptide cleavage sites in proteins</li></ul>
<ul class="row"><li>esignalp</li><li>Report protein signal cleavage sites</li></ul>
<ul class="row"><li>eyinoyang</li><li>Report O-(beta)-GlcNAc attachment sites</li></ul>
</ul>
<li class="group_title">Protein motifs &#8212; Protein motif searches</li>
<ul class="group">
<ul class="row"><li>antigenic</li><li>Find antigenic sites in proteins</li></ul>
<ul class="row"><li>epestfind</li><li>Find PEST motifs as potential proteolytic cleavage sites</li></ul>
<ul class="row"><li>fuzzpro</li><li>Search for patterns in protein sequences</li></ul>
<ul class="row"><li>fuzztran</li><li>Search for patterns in protein sequences (translated)</li></ul>
<ul class="row"><li>patmatdb</li><li>Search protein sequences with a sequence motif</li></ul>
<ul class="row"><li>patmatmotifs</li><li>Scan a protein sequence with motifs from the PROSITE database</li></ul>
<ul class="row"><li>preg</li><li>Regular expression search of protein sequence(s)</li></ul>
<ul class="row"><li>pscan</li><li>Scan protein sequence(s) with fingerprints from the PRINTS database</li></ul>
<ul class="row"><li>sigcleave</li><li>Report on signal cleavage sites in a protein sequence</li></ul>
<ul class="row"><li>elipop</li><li>Predict lipoproteins</li></ul>
<ul class="row"><li>eiprscan</li><li>Motif detection</li></ul>
<ul class="row"><li>omeme</li><li>Motif detection</li></ul>
<ul class="row"><li>emast</li><li>Motif detection</li></ul>
<ul class="row"><li>ememe</li><li>Multiple EM for motif elicitation</li></ul>
<ul class="row"><li>ememetext</li><li>Multiple EM for motif elicitation, text file only</li></ul>
</ul>
<li class="group_title">Protein mutation &#8212; Protein sequence mutation</li>
<ul class="group">
<ul class="row"><li>msbar</li><li>Mutate a sequence</li></ul>
<ul class="row"><li>shuffleseq</li><li>Shuffle a set of sequences maintaining composition</li></ul>
</ul>
<li class="group_title">Protein profiles &#8212; Protein profile generation and searching</li>
<ul class="group">
<ul class="row"><li>profit</li><li>Scan one or more sequences with a simple frequency matrix</li></ul>
<ul class="row"><li>prophecy</li><li>Create frequency matrix or profile from a multiple alignment</li></ul>
<ul class="row"><li>prophet</li><li>Scan one or more sequences with a Gribskov or Henikoff profile</li></ul>
<ul class="row"><li>seqsearch</li><li>Generate PSI-BLAST hits (DHF file) from a DAF file</li></ul>
<ul class="row"><li>siggen</li><li>Generate a sparse protein signature from an alignment</li></ul>
<ul class="row"><li>siggenlig</li><li>Generate ligand-binding signatures from a CON file</li></ul>
<ul class="row"><li>sigscan</li><li>Generate hits (DHF file) from a signature search</li></ul>
<ul class="row"><li>sigscanlig</li><li>Search ligand-signature library and writes hits (LHF file)</li></ul>
</ul>
<li class="group_title">Protein properties &#8212; Protein physicochemical properties</li>
<ul class="group">
<ul class="row"><li>charge</li><li>Draw a protein charge plot</li></ul>
<ul class="row"><li>hmoment</li><li>Calculate and plot hydrophobic moment for protein sequence(s)</li></ul>
<ul class="row"><li>iep</li><li>Calculate the isoelectric point of proteins</li></ul>
<ul class="row"><li>octanol</li><li>Draw a White-Wimley protein hydropathy plot</li></ul>
<ul class="row"><li>pepinfo</li><li>Plot amino acid properties of a protein sequence in parallel</li></ul>
<ul class="row"><li>pepstats</li><li>Calculate statistics of protein properties</li></ul>
<ul class="row"><li>pepwindow</li><li>Draw a hydropathy plot for a protein sequence</li></ul>
<ul class="row"><li>pepwindowall</li><li>Draw Kyte-Doolittle hydropathy plot for a protein alignment</li></ul>
</ul>
<li class="group_title">Test &#8212; Testing tools, not for general use.</li>
<ul class="group">
<ul class="row"><li>crystalball</li><li>Answer every drug discovery question about a sequence</li></ul>
</ul>
<li class="group_title">Utils &#8212; Utility tools</li>
<ul class="group">
<ul class="row"><li>embossdata</li><li>Find and retrieve EMBOSS data files</li></ul>
<ul class="row"><li>embossupdate</li><li>Checks for more recent updates to EMBOSS</li></ul>
<ul class="row"><li>embossversion</li><li>Report the current EMBOSS version number</li></ul>
</ul>
<li class="group_title">Utils database creation &#8212; Database installation</li>
<ul class="group">
<ul class="row"><li>aaindexextract</li><li>Extract amino acid property data from AAINDEX</li></ul>
<ul class="row"><li>cutgextract</li><li>Extract codon usage tables from CUTG database</li></ul>
<ul class="row"><li>jaspextract</li><li>Extract data from JASPAR</li></ul>
<ul class="row"><li>printsextract</li><li>Extract data from PRINTS database for use by pscan</li></ul>
<ul class="row"><li>prosextract</li><li>Process the PROSITE motif database for use by patmatmotifs</li></ul>
<ul class="row"><li>rebaseextract</li><li>Process the REBASE database for use by restriction enzyme applications</li></ul>
<ul class="row"><li>tfextract</li><li>Process TRANSFAC transcription factor database for use by tfscan</li></ul>
<ul class="row"><li>cathparse</li><li>Generate DCF file from raw CATH files</li></ul>
<ul class="row"><li>domainnr</li><li>Remove redundant domains from a DCF file</li></ul>
<ul class="row"><li>domainseqs</li><li>Add sequence records to a DCF file</li></ul>
<ul class="row"><li>domainsse</li><li>Add secondary structure records to a DCF file</li></ul>
<ul class="row"><li>scopparse</li><li>Generate DCF file from raw SCOP files</li></ul>
<ul class="row"><li>ssematch</li><li>Search a DCF file for secondary structure matches</li></ul>
<ul class="row"><li>allversusall</li><li>Sequence similarity data from all-versus-all comparison</li></ul>
<ul class="row"><li>seqnr</li><li>Remove redundancy from DHF files</li></ul>
<ul class="row"><li>domainer</li><li>Generate domain CCF files from protein CCF files</li></ul>
<ul class="row"><li>hetparse</li><li>Convert heterogen group dictionary to EMBL-like format</li></ul>
<ul class="row"><li>pdbparse</li><li>Parse PDB files and writes protein CCF files</li></ul>
<ul class="row"><li>pdbplus</li><li>Add accessibility and secondary structure to a CCF file</li></ul>
<ul class="row"><li>pdbtosp</li><li>Convert swissprot:PDB codes file to EMBL-like format</li></ul>
<ul class="row"><li>sites</li><li>Generate residue-ligand CON files from CCF files</li></ul>
</ul>
<li class="group_title">Utils database indexing &#8212; Database indexing</li>
<ul class="group">
<ul class="row"><li>dbiblast</li><li>Index a BLAST database</li></ul>
<ul class="row"><li>dbifasta</li><li>Index a fasta file database</li></ul>
<ul class="row"><li>dbiflat</li><li>Index a flat file database</li></ul>
<ul class="row"><li>dbigcg</li><li>Index a GCG formatted database</li></ul>
<ul class="row"><li>dbxcompress</li><li>Compress an uncompressed dbx index</li></ul>
<ul class="row"><li>dbxedam</li><li>Index the EDAM ontology using b+tree indices</li></ul>
<ul class="row"><li>dbxfasta</li><li>Index a fasta file database using b+tree indices</li></ul>
<ul class="row"><li>dbxflat</li><li>Index a flat file database using b+tree indices</li></ul>
<ul class="row"><li>dbxgcg</li><li>Index a GCG formatted database using b+tree indices</li></ul>
<ul class="row"><li>dbxobo</li><li>Index an obo ontology using b+tree indices</li></ul>
<ul class="row"><li>dbxreport</li><li>Validate index and report internals for dbx databases</li></ul>
<ul class="row"><li>dbxresource</li><li>Index a data resource catalogue using b+tree indices</li></ul>
<ul class="row"><li>dbxstat</li><li>Dump statistics for dbx databases</li></ul>
<ul class="row"><li>dbxtax</li><li>Index NCBI taxonomy using b+tree indices</li></ul>
<ul class="row"><li>dbxuncompress</li><li>Uncompress a compressed dbx index</li></ul>
</ul>

</ul>

</div>
</div>
</body>
</html>

